### PR TITLE
Cleanup.

### PR DESCRIPTION
**Pull request by geeknoid**
- Delete dead code.

- Rename a bunch of *2 symbols to just *.

### CODE REVIEW

File: mixer/pkg/config/store/store_test.go [hunk 1]
```diff
@@ -47,16 +47,18 @@ func (t *testStore) Watch(ctx context.Context) (<-chan BackendEvent, error) {
...
-		return &testStore{memstore: *createMemstore(u)}, nil
+func registerTestStore(builders map[string]StoreBuilder) {
+	builders["test"] = func(u *url.URL) (StoreBackend, error) {
+		return &testStore{
+			memstore: *newMemstore(u).(*memstore),
+		}, nil
 	}
 }
 
 func TestStore2(t *testing.T) {
```
Comment (jmuk) [inline mixer/pkg/config/store/store_test.go]: can you rename the function name as `TestStore`?

File: mixer/pkg/config/store/store_test.go [hunk 2]
```diff
@@ -113,8 +115,8 @@ func TestStore2(t *testing.T) {
 }
 
 func TestStore2WatchMultiple(t *testing.T) {
```
Comment (jmuk) [inline mixer/pkg/config/store/store_test.go]: ditto, `TestStoreWatchMultiple`?

File: mixer/pkg/config/store/store_test.go [hunk 3]
```diff
@@ -138,12 +140,12 @@ func TestStore2WatchMultiple(t *testing.T) {
 }
 
 func TestStore2Fail(t *testing.T) {
```
Comment (jmuk) [inline mixer/pkg/config/store/store_test.go]: ditto

File: mixer/pkg/config/store/store_test.go [hunk 4]
```diff
@@ -171,7 +173,7 @@ func TestStore2Fail(t *testing.T) {
 }
 
 func TestRegistry2(t *testing.T) {
```
Comment (jmuk) [inline mixer/pkg/config/store/store_test.go]: `TestRegistry`?

**Comment by geeknoid** [comment]
Feedback addressed.

PTAL.

**Comment by jmuk** [comment]
Please also update `install/kubernetes/templates/istio-mixer.yaml.tmpl` regarding the flag name change.

**Comment by geeknoid** [comment]
OK, I changed all references to --configStore2URL to --configStoreURL.

**Pull request merged**